tests/compile-fail/non_extern_foreign_call/clashing.rs

```diff
@@ -0,0 +1,20 @@
+// revisions: two three
+
+#[no_mangle]
+fn foo() {}
+
+#[export_name = "foo"]
+fn bar() {}
+
+#[cfg(three)]
+#[export_name = "foo"]
+fn baz() {}
+
+fn main() {
+    extern "Rust" {
+        fn foo();
+    }
+    unsafe { foo() }
+    //[two]~^ ERROR unsupported operation: multiple definitions of `foo`
+    //[three]~^^ ERROR unsupported operation: multiple definitions of `foo`
+}
```

Comment:
I hope I can add a multi-crate version of this (`compile-fail`) test, but `aux-build` is not working -- it seems to ignore `rustc-env: MIRI_BE_RUSTC=target` in the `aux-build` file.

Reply:
Yeah the aux stuff never worked in the Miri test suite. :/

Reply:
See https://github.com/rust-lang/miri/issues/622